Adyen/UI/List/ListCell.swift

```diff
@@ -97,7 +97,7 @@ public final class ListCell: UITableViewCell {
         let constraints = [
             itemView.topAnchor.constraint(equalTo: topAnchor),
             itemView.leadingAnchor.constraint(equalTo: layoutGuide.leadingAnchor),
-            itemView.trailingAnchor.constraint(lessThanOrEqualTo: layoutGuide.trailingAnchor),
+            itemView.trailingAnchor.constraint(equalTo: layoutGuide.trailingAnchor),
```

Comment:
I am wondering why we had it like this in a first place 🤔

Reply:
I don't know, but I think its not needed

Reply:
Something on iOS 10 or 11, maybe?